src/Apps/Artist/Components/ArtistCollectionsRail/ArtistCollectionEntity.tsx

```diff
@@ -38,9 +38,11 @@ export class ArtistCollectionEntity extends React.Component<CollectionProps> {
     } = this.props.collection
     const artworks = artworks_connection.edges.map(({ node }) => node)
     const formattedTitle = (title && title.split(": ")[1]) || title
-    const bgImages = compact(artworks.map(({ image }) => image && image.url))
+    const bgImages = compact(
+      artworks.map(({ image }) => image && image.resized.url)
```

Comment:
I don't know if our back-end services prevent us from having an undefined `image.resized` from this query, but we should be more cautious just in case.

Two options come to mind.

1. Chain another `&&`:

```
const bgImages = compact(
      artworks.map(({ image }) => image && image.resized && image.resized.url)
```

2. Use the `get` helper, which takes the top level object (`image`), and safely traverses a path to a property.

import the helper:

```
import { get } from "Utils/get"
```

and use it like this: 

```
const bgImages = compact(
      artworks.map(({ image }) => get(image, i => i.resized.url))
```

[Example](https://github.com/artsy/reaction/blob/0d6255c76828a43621067bdfa0c62c7c9b503d49/src/Apps/Artist/Routes/Overview/index.tsx#L79)

Reply:
Thanks for the comment. @pepopowitz  @damassi 
I did not take the "undefined" case into account when I was working on the ticket. 

Also, I have a question about the changing you suggested. 
On this one,
```javascript
const bgImages = compact(
      artworks.map(({ image }) => image && image.resized && image.resized.url)
```
Why do you ask to chain `image.resized`?  
In my understanding, for `image && image.resized && image.resized.url`, if one of these return `undefined`, the final result will get `undefined`, then we used `compact` from lodash to remove the `undefined` or `null`. We will get an empty array. On line 54, if the array were empty, we used the header image in the carousel.

Reply:
In the scenario I'm thinking of, we would not get to line 54, because we'd hit a `Cannot read property 'url' of undefined` exception. 

The existing code in the PR works great for artworks that have one of these shapes:

```
{
  "image": undefined
}

{
  "image": {
    "resized": undefined
  }
}

{
  "image": {
    "resized": "some-url"
  }
}
```

But, if the shape looks like this next example, the code would throw an exception:

```
{
  "image": { }
}
```

Because `image.resized` is undefined, and we'd be trying to reference a property on undefined. Adding the `&& image.resized ` in the middle of your mapping predicate will protect us from getting those exceptions.

I don't know that our back-end services would ever return that shape....but it doesn't hurt us at all to be cautious about it.

Reply:
Oh, I see. It makes sense.
Thank you for explaining. 🤝 🙇